sdk/keyvault/keyvault-keys/README.md

```diff
@@ -21,7 +21,14 @@ Use the client library for Azure KeyVault Keys in your Node.js application to
 - Get the versions of a key.
 - As well as obtaining the attributes of a key.
 
-**Please Note:** This is a preview version of the KeyVault Keys library and does not yet have the full capability of the Keys API. Currently missing are methods related to crypto capabilities: encrypt, decrypt, sign, verify, wrap, and unwrap. These are expected to be added in the next release.
```

Comment:
I believe we should leave this somewhere, since it is still true.

Reply:
This message is specifically about missing functionality between track 2 and track 1. We've now added that functionality back.

Reply:
True! I was referring to the "this is a preview version..." <-- which is still true.

Reply:
Added